rspace/src/test/scala/coop/rchain/rspace/TestImplicitHelpers.scala

```diff
@@ -0,0 +1,14 @@
+package coop.rchain.rspace
+import cats.Id
+import org.scalatest.enablers.Definition
+
+//noinspection ConvertExpressionToSAM
+trait TestImplicitHelpers {
+  // Some helpers for usage only in the tests -- save us A LOT of explicit casting from Either to Option
+  // it is safe because left type of `Either` is `Nothing` -- we don't expect any invalid states from the matcher
+  implicit def eitherDefinitionScalatest[E, A]: Definition[Id[Either[E, Option[A]]]] =
+    new Definition[Id[Either[E, Option[A]]]] {
+      override def isDefined(thing: Id[Either[E, Option[A]]]): Boolean =
```

Comment:
Is this used anywhere?

Reply:
Yes, everywhere there is `_sth_ isDefined` where type of `_sth_` is `Id[Either[Nothing, Option[_]]]` .